sql-plugin/src/main/scala/com/nvidia/spark/rapids/CostBasedOptimizer.scala

```diff
@@ -308,6 +312,10 @@ class GpuCostModel(conf: RapidsConf) extends CostModel {
   }
 
   private def exprCost[INPUT <: Expression](expr: BaseExprMeta[INPUT], rowCount: Double): Double = {
+    if (MemoryCostHelper.isWindowExpr(expr)) {
+      // Window expressions are Unevaluable and accessing dataType causes an exception
+      return 0
```

Comment:
Can we use `0d` for consistency with the uses of zero just below it.  I think`0.0` is actually more Scala-esque but consistency is most important.